engine/runtime/src/test/scala/org/enso/compiler/test/codegen/AstToIrTest.scala

```diff
@@ -890,13 +890,14 @@ class AstToIrTest extends CompilerTest with Inside {
   "AST translation for imports and exports" should {
     "properly support different kinds of imports" in {
       val imports = List(
-        "import Foo.Bar as Baz",
-        "import Foo.Bar",
-        "from Foo.Bar import Baz",
-        "from Foo.Bar import Baz, Spam",
-        "from Foo.Bar import all",
-        "from Foo.Bar as Eggs import all hiding Spam",
-        "from Foo.Bar import all hiding Spam, Eggs"
+        "import username.Foo.Bar as Baz",
+        "import project.Foo.Bar",
+        "from project import all",
+        "from Foo.Bar.Quux import Baz",
+        "from Foo.Bar.Test import Baz, Spam",
```

Comment:
Why these two are not prefixed by username? Or is `Foo` meant to be the username here? It's a bit confusing as in other examples `Foo` is used as project/module name. If this is meant to test capitalised usernames, I'd suggest making this `Username` instead of `Foo`.